information-schema/information-schema-runaway-watches.md

```diff
@@ -0,0 +1,114 @@
+---
+title: RUNAWAY_WATCHES
+summary: 了解 information_schema 表 `RUNAWAY_WATCHES`。
+---
+
+# RUNAWAY_WATCHES
+
+`RUNAWAY_WATCHES` 表展示 Runaway Queries 监控列表，见 [Runaway Queries](/tidb-resource-control.md#管理资源消耗超出预期的查询-runaway-queries)。
+
+```sql
+USE information_schema;
+DESC runaway_watches;
+```
+
+```sql
++---------------------+--------------+------+------+---------+-------+
+| Field               | Type         | Null | Key  | Default | Extra |
++---------------------+--------------+------+------+---------+-------+
+| ID                  | bigint(64)   | NO   |      | NULL    |       |
+| RESOURCE_GROUP_NAME | varchar(32)  | NO   |      | NULL    |       |
+| START_TIME          | varchar(32)  | NO   |      | NULL    |       |
+| END_TIME            | varchar(32)  | YES  |      | NULL    |       |
+| WATCH               | varchar(12)  | NO   |      | NULL    |       |
+| WATCH_TEXT          | text         | NO   |      | NULL    |       |
+| SOURCE              | varchar(128) | NO   |      | NULL    |       |
+| ACTION              | varchar(12)  | NO   |      | NULL    |       |
++---------------------+--------------+------+------+---------+-------+
+8 rows in set (0.00 sec)
+```
+
+> **警告：**
+>
+> 该功能目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+
+## 示例
+
+```sql
+select * from information_schema.runaway_watches\G; -- 查询 Runaway Queries 识别名单
+```
+
+```sql
+*************************** 1. row ***************************
+                 ID: 20003
+RESOURCE_GROUP_NAME: rg2
+         START_TIME: 2023-07-28 13:06:08
+           END_TIME: UNLIMITED
+              WATCH: Similar
+         WATCH_TEXT: 5b7fd445c5756a16f910192ad449c02348656a5e9d2aa61615e6049afbc4a82e
+             SOURCE: 127.0.0.1:4000
+             ACTION: Kill
+*************************** 2. row ***************************
+                 ID: 16004
+RESOURCE_GROUP_NAME: rg2
+         START_TIME: 2023-07-28 01:45:30
+           END_TIME: UNLIMITED
+              WATCH: Similar
+         WATCH_TEXT: 3d48fca401d8cbb31a9f29adc9c0f9d4be967ca80a34f59c15f73af94e000c84
+             SOURCE: 127.0.0.1:4000
+             ACTION: Kill
+2 rows in set (0.00 sec)
+```
+
+```sql
+query watch add resource group rg1 sql text exact to 'select * from sbtest.sbtest1'; -- 添加识别名单
+```
+
+```sql
+select * from information_schema.runaway_watches\G; -- 查询 Runaway Queries 识别名单
+```
```

Comment:
````suggestion
添加一个识别项到资源组 `rg1`：

```sql
QUERY WATCH ADD RESOURCE GROUP rg1 SQL TEXT EXACT TO 'select * from sbtest.sbtest1';
```

再次查询 Runaway Queries 识别名单：

```sql
SELECT * FROM INFORMATION_SCHEMA.RUNAWAY_WATCHES\G;
```

输出结果如下：
````